apps/prairielearn/src/ee/pages/instructorAiGenerateQuestion/instructorAiGenerateQuestion.html.ts

```diff
@@ -44,6 +44,12 @@ export function AiGeneratePage({ resLocals }: { resLocals: Record<string, any> }
                 </button>
               </form>
               <div id="generation-results"></div>
+              <br />
+              <div>
+                <form action="${resLocals.urlPrefix}/ai_generate_question_audit">
+                  <button type="submit" class="btn btn-primary">Previous Generation Jobs</button>
+                </form>
```

Comment:
Why is this a form? A link (`<a>`) would almost certainly suffice.

Reply:
changed in 3c8bafd1470728a8c79b488213de1dfae0aed79e.